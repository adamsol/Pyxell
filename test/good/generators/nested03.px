
func* f(n: Int = 0): Int def
    i: Int

    func* g(): Int def
        for x in i...n do
            yield x

    for i in 0..n do
        for j in g() do
            yield j * 2

print [...f()]
print [...f(4)]
