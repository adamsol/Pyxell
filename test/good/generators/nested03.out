[]
[0, 2, 4, 6, 2, 4, 6, 4, 6, 6]
